crypto/asn1/tasn_fre.c

```diff
@@ -89,7 +89,8 @@ static void asn1_item_embed_free(ASN1_VALUE **pval, const ASN1_ITEM *it,
 
     case ASN1_ITYPE_NDEF_SEQUENCE:
     case ASN1_ITYPE_SEQUENCE:
-        if (asn1_do_lock(pval, -1, it) > 0)
+        count = asn1_do_lock(pval, -1, it);
```

Comment:
what's the point of adding the 'count' variable?  and is changing the test the right thing?